src/components/templates/AddStory.js

```diff
@@ -23,13 +23,24 @@ const AddStory = ({ toggleModal }) => {
       !newStory.title.trim()
       || !newStory.content.trim()
     ) return;
-
+    //don't attempt if the user is not logged in
+    if (
+      !currentUser
+    ) return;
```

Comment:
We can just make this a single line. I only break it down into multiple lines if you're chaining more than one condition in a single if statement 👍🏽